experimental/babel-preset-env/test/fixtures/preset-options-add-used-built-ins/builtins-used-evaluated-variables/actual.js

```diff
@@ -0,0 +1,13 @@
+var arrayInstance = [];
+var objectClass = Object;
+var assignStr = "assign";
+var valuesStr = "values";
+var inclidesStr = "includes";
+var findStr = "find";
+
+// Allow instance methods be assigned to variables.
+arrayInstance[inclidesStr](); // include
```

Comment:
Should we split these up into individual tests so that we can make sure they each include (or not include)?